sixtyfps_compiler/passes/binding_analysis.rs

```diff
@@ -210,7 +210,7 @@ fn visit_implicit_layout_info_dependencies(
         }
         "Text" => {
             vis(&NamedReference::new(&item, "text"));
-            vis(&NamedReference::new(&item, "font_familly"));
+            vis(&NamedReference::new(&item, "font_family"));
```

Comment:
this one is interesting!!  that was a bug!

Reply:
My first bug fix then! :-)

Reply:
Slow and steady wins the race :-)

btw, did you configure the spell checker in any special way?

Reply:
I did add a ton of words to its ignore/workspace list, yes.

You have quite a few apprevtions and shorthand forms in your codebase that the code spellchecker does not like too much!

Reply:
I'm thinking of adding my `cspell.json` to the git repo simply. Or you could also submit yours if you have one already :-)

Reply:
In my experience the list is manageable -- if you do are comfortable to have really short (<= 3 chars) or really long (real words with camelcase/snakecase) local variables.